src/shared/modules/cypher/cypherDuck.ts

```diff
@@ -256,7 +256,7 @@ export const handleForcePasswordChangeEpic = (some$: any, store: any) =>
       ) => {
         if (!action.$$responseChannel) return Rx.Observable.of(null)
 
-        return new Promise((resolve, _reject) => {
+        return new Promise(resolve => {
```

Comment:
a little concerning (maybe?) that we're not handling the error here, but that's out of scope for this PR